internal/cli/option/flag.go

```diff
@@ -70,10 +95,29 @@ func FreightAlias(fs *pflag.FlagSet, stage *string, usage string) {
 	fs.StringVar(stage, FreightAliasFlag, "", usage)
 }
 
+// Git adds the GitFlag to the provided flag set.
+func Git(fs *pflag.FlagSet, git *bool, usage string) {
+	fs.BoolVar(git, GitFlag, false, usage)
+}
+
+// Helm adds the HelmFlag to the provided flag set.
+func Helm(fs *pflag.FlagSet, helm *bool, usage string) {
+	fs.BoolVar(helm, HelmFlag, false, usage)
+}
+
+// Image adds the ImageFlag to the provided flag set.
+func Image(fs *pflag.FlagSet, image *bool, usage string) {
+	fs.BoolVar(image, ImageFlag, false, usage)
+}
+
 func InsecureTLS(fs *pflag.FlagSet, opt *Option) {
 	fs.BoolVar(&opt.InsecureTLS, "insecure-skip-tls-verify", false, "Skip TLS certificate verification")
 }
 
+func InteractivePassword(fs *pflag.FlagSet, changePasswordInteractively *bool, usage string) {
```

Comment:
```suggestion
// InteractivePassword adds the InteractivePasswordFlag to the provided flag set.
func InteractivePassword(fs *pflag.FlagSet, changePasswordInteractively *bool, usage string) {
```

Reply:
Fixed in a051595b97541cab94bb7e8f3be72f480a7f8022